apps/plato_single_variant/dxapp.json

```diff
@@ -137,32 +156,45 @@
       "label": "Correction Type",
       "class": "array:string",
       "choices": [ "Bonferoni", "FDR" ],
-      "optional": true
+      "optional": true,
+      "help": "(Optional) A string, with the description on which of multiple test correction strategies to use in reporting results from the regression."
     }
   ],
   "outputSpec": [
     {
       "name": "output_files",
       "label": "Output files",
-      "class": "array:file"
+      "class": "array:file",
+      "patterns": ["*"],
+      "optional": false,
+      "help": "A list of output files."
     }
   ],
+  "regionalOptions": {
+    "aws:us-east-1": {"systemRequirements": {"main": {"instanceType": "mem1_ssd1_x2"},
+                                             "*": {"instanceType": "mem1_ssd1_x32"}},
+                      "assetDepends": [{"name": "marylyn_asset", "id": "record-FGpf7zQ0kfbZxj6q4ZzQ2BPG"},
+                                       {"name": "boost_1_48_0_asset", "id": "record-FGzQz500fZ0pQgXJ2xqbk3xz"}]},
```

Comment:
Should boost go in the common asset?

Reply:
This app (Plato) and Biobin require specific boost version 1.48.0, which is default on our Ubuntu 12.04 machine but missing in our Ubuntu 14.04 machines. The 14.04 machine can come with a later version of boost.
I thought since only the above two apps require this specific and old boost version, it is okay to just add the asset to the two apps.